### ISSUE TITLE
Write about CSUA Artifacts!

### ISSUE THREAD

**Issue by robertquitt** [CSUA/documentaries#1]
Artifacts:

- The Educator
- Application form
- Beasty trident

**Comment by mark64** [comment]
Can __you__ __**post**__ **the** ~~full~~ [list](list) `from` ![the](the.png "the") whiteboard?

**Comment by robertquitt** [comment]
@mark64 Yeah it's in [jargon.md](https://github.com/CSUA/documentaries/blob/master/jargon.md)
